apps/src/lib/kits/maker/boards/microBit/ExternalLed.js

```diff
@@ -19,4 +18,13 @@ export default class ExternalLed {
   toggle() {
     return this.isOn ? this.off() : this.on();
   }
+
+  blink(delay) {
+    this.off();
+    setInterval(this.toggle_.bind(this), delay);
+  }
+
+  toggle_() {
+    return this.isOn ? this.off() : this.on();
+  }
```

Comment:
Do you need both `toggle()` and `toggle_()` if they're doing the same thing?

Reply:
I originally called on `toggle` in `blink`, but got some strange behavior with the LedScreen. But I think it may have been an unrelated issue! I actually think I need to keep this version of `toggle_` to address your other question about how to turn `blink` off. Thanks!